iroha/src/account.rs

```diff
@@ -11,7 +11,7 @@ pub struct Account {
     /// An Identification of the `Account`.
     pub id: Id,
     /// Asset's in this `Account`.
-    pub assets: BTreeMap<<Asset as Identifiable>::Id, Asset>,
+    pub components: BTreeMap<<Component as Identifiable>::Id, Component>,
```

Comment:
I think it's better to use fully qualified name like `asset:Component` here or add an import alias `AssetComponent` otherwise it's difficult to understand what component this is.

Reply:
Maybe even name the field `asset_components`.

Reply:
I don't really think the name component is good. assets is much better and makes sense. Domain has asset definition and accounts have assets. It seems clear to me that this is how it should be.

Reply:
I would just keep asset everywhere in account.rs and not use component.

Reply:
okay, so I may rename `Asset` to `AssetDefinition` and `Component` to the `Asset`?